test/public/flps/overview.test.js

```diff
@@ -152,23 +165,20 @@ module.exports = () => {
         const oldFirstRowId = await getFirstRow(table, page);
         const secondPage = await page.$('#page2');
         await secondPage.evaluate((button) => button.click());
-        await waitForTimeout(500);
```

Comment:
You don't wait for the table to be redrawn, you need to explicit a table length here

Reply:
This test is skipped for some reason, I was going to investigate why and implement the real functionality at some point but I just got rid of the timeout

Reply:
I did not noticed. Could you delete it then?